src/testapp/main.cpp

```diff
@@ -308,6 +308,7 @@ void start_magnum_async()
         g_testApp.close_session(g_testApp.m_windowApp);
 
         OSP_LOG_INFO("Closed Magnum Application");
+        std::exit(0);
```

Comment:
i'm fairly certain we would want to keep the behavior of a "graceful" exit.

if that's not working, we should debug why and get it fixed.

Reply:
I think this was for debugging something. Will remove.

Reply:
Hm, if I remove it and close the window it doesn't stop the application. So I think exiting here is more expected behavior.

Reply:
closing the window isn't supposed to exit the application in the current code.

Arguably that's not really what most people want, but it's what the current code was intended to do.